.github/workflows/cd.yml

```
@@ -0,0 +1,246 @@
+        run: docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
+
+  # --- Job 2: EC2에 Blue-Green 배포 ---
+  deploy-blue-green:
+    name: Deploy to EC2 (Blue-Green)
+    needs: build-and-push # 'build-and-push' job이 성공해야만 실행됨
+    runs-on: ubuntu-latest
+
+    steps:
+      # 1. AWS 자격 증명 설정 (EC2에서 ECR 접근 등에 필요할 수 있음)
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 92: did not find expected ',' or '}'

(syntax-check)